regression/cbmc/Quantifiers1/quantifier-with-side-effect.desc

```diff
@@ -1,8 +1,8 @@
 CORE
 quantifier-with-side-effect.c
 
-^EXIT=6$
+^EXIT=134$
 ^SIGNAL=0$
-^file .* line 10 function main: quantifier must not contain side effects$
+^Invariant check failed
+^Reason: quantifier must not contain side effects
```

Comment:
You'll have to add this one to `regression/validate-trace-xml-schema/check.py` as the failing invariant check implies the overall output isn't valid XML. But if you pick up my suggestion below then we're back with `EXIT=6` and you just need to update the error message ("quantifier must not contain function call").